templates/zerver/create_realm.html

```diff
@@ -39,13 +39,6 @@ <h1 class="get-started">{{ _("Create a new Zulip organization") }}</h1>
                     </div>
                 </form>
             </div>
-            <div class="bottom-text">
-                {% trans %}
-                Or import
-                from <a href="/help/import-from-slack">Slack</a>, <a href="/help/import-from-mattermost">Mattermost</a>,
-                or <a href="/help/import-from-rocketchat">Rocket.Chat</a>.
-                {% endtrans %}
-            </div>
```

Comment:
We might want to keep this if we're keeping this feature hidden behind a `development_environment` check to make iteration a bit easier; I kinda want to merge it with the production UI unchanged if that's easy.